Bookish/Bookish.Web/Views/Home/AddedBook.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@model Bookish.Web.Models.AddedBookViewModel
+@{
+    ViewData["Title"] = "AddedBook";
+}
+
+<h1>@Model.Title has been successfully added! Here are your barcodes</h1>
+
+
+@foreach (var barcodeId in Model.BarcodeIds)
+    {
+        <img src="@($"../images/barcodes/barcode{barcodeId}.png")" class="rounded mx-auto padded-top"/>
```

Comment:
I'd recommend having each barcode on a new line 🙂

Reply:
I was trying many different css things to get the barcodes to display in the centre. The current thing in there is bootstrap code that is supposed to do the same but for some reason it doesn't work :(

Reply:
No matter what methods I try they seem to just do this
![image](https://user-images.githubusercontent.com/44266225/98133032-006d2980-1eb5-11eb-9ba5-62859b65c294.png)

Reply:
Is this any help: https://stackoverflow.com/questions/7055393/center-image-using-text-align-center? 🙂